zhaquirks/sengled/e1e-g7f.py

```diff
@@ -0,0 +1,195 @@
+"""Sengled E1E-G7F device."""
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    PollControl,
+    PowerConfiguration,
+)
+
+from .. import Bus
+from ..const import (
+    ARGS,
+    COMMAND,
+    COMMAND_OFF,
+    COMMAND_ON,
+    COMMAND_STEP,
+    DEVICE_TYPE,
+    DIM_DOWN,
+    DIM_UP,
+    DOUBLE_PRESS,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    LONG_PRESS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    SHORT_PRESS,
+    TURN_OFF,
+    TURN_ON,
+    ZHA_SEND_EVENT,
+)
+
+
+class SengledE1EG7FOnOffCluster(CustomCluster, OnOff):
+    """Sengled E1E-G7F OnOff cluster."""
+
+    cluster_id = OnOff.cluster_id
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+
+        super().__init__(*args, **kwargs)
+
+        self.endpoint.device.on_off_bus.add_listener(self)
+
+
+class SengledE1EG7FLevelControlCluster(CustomCluster, LevelControl):
+    """Sengled E1E-G7F LevelControl cluster."""
+
+    cluster_id = LevelControl.cluster_id
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+
+        super().__init__(*args, **kwargs)
+
+        self.endpoint.device.level_control_bus.add_listener(self)
+
+
+class SengledE1EG7FManufacturerSpecificCluster(CustomCluster):
+    """Sengled E1E-G7F manufacturer-specific cluster."""
+
+    cluster_id = 0xFC10
```

Comment:
let's add `name` and `ep_attribute` attributes for this cluster. Feel free to adjust to your likings

```suggestion
    cluster_id = 0xFC10
    name = "Sengled Manufacturer Specific"
    ep_attribute = "sengled_manufacturer"
```

Other than that looks good.

Reply:
Will do, but can you explain what the purpose of those is? Where will these values ever show up or be used? I was not able to find any documentation about them when I looked before.

Reply:
this is mostly to make it future proof. ZHA uses those internally and expect all clusters to have it.

Reply:
Done